servlet/src/main/java/io/undertow/servlet/spec/ServletCookieAdaptor.java

```diff
@@ -159,14 +161,14 @@ public Cookie setComment(final String comment) {
 
     @Override
     public boolean isSameSite() {
-        return sameSite;
+        return this.sameSiteMode != null;
     }
 
-    @Override
-    public Cookie setSameSite(final boolean sameSite) {
-        this.sameSite = sameSite;
-        return this;
-    }
+//    @Override
```

Comment:
This and other instances should maybe be just turned into dummy methods with @Depracated?
Im fairly sure samesite might have worked if used on certain layer, its just integration was broken.